benchmarks/src/main/java/io/grpc/benchmarks/qps/OpenLoopClient.java

```diff
@@ -202,7 +202,7 @@ public void onCompleted() {
 
     private void waitForRpcsToComplete(int duration) {
       long now = System.nanoTime();
-      long end = now + duration * 1000 * 1000 * 1000;
+      long end = now + duration * 1000L * 1000 * 1000;
```

Comment:
Doesn't this still have implicit conversion? I feel like this is style, especially since `now` is the only thing there that is likely to overflow 32 bits so the long isn't significant. The one for sleep is also unclear to me why it is helpful.

Reply:
I agree that it isn't useful and makes the code uglier, so reverting it.

Reply:
To be clear, I wouldn't mind if someone did this as I see it as personal style. But that's also why I'd leave it as-is unless it was hurting something.